src/components/mortgage-investment-compare/index.js

```diff
@@ -49,17 +61,41 @@ class MortgageInvestmentCompare extends React.Component {
 
     const result = this.getResult(input)
```

Comment:
I had to move this to `componentWillMount` because I was having some sort of race condition that prevented me from  reading `includeInflation` and `includeROI` from the state when launching the app. Feel free to pull locally and revert the change if you'd like to dig further into why this was happening.